### ISSUE TITLE
Keep caret position after reformatting

### ISSUE THREAD

**Issue by koeberlue** [jucham/clean-code-method-rearranger#2]
After reformatting the code the caret always jumps to the end of the class. This makes it harder to work for me because I usually to reformat multiple times while editing a single file and after each reformat i need to search for the method i was working on.

Would it be possible to set the cursor to the same position it was before reformatting. For me this would probably be the most intuitive behavior:

| Caret before reformatting | Caret after reformatting |
| ------------- |-------------|
| inside method | inside the same method on the same line |
| between two methods | on top of the second method |

So basically the caret should always move with the method it "belongs" to when reformatting

Please let me know if i you need any further information and thanks for the great work

**Comment by jucham** [comment]
Thanks for pointing out the problem and sorry for delaying this a bit. A new release fixing this is available. Hoping it's satisfying for you.
